flocker/cli/functional/test_sshconfig.py

```diff
@@ -64,7 +64,8 @@ def test_connection_failed(self):
 
         exc = self.assertRaises(CalledProcessError,
                                 self.configure_ssh, b"127.0.0.1", port)
-        self.assertIn(b"refused", exc.output)
+        self.assertTrue(b"refused" in exc.output or "Operation timed out" in
+                        exc.output)
```

Comment:
I guess a bound socket that isn't listening behaves differently on OSX. I guess there should be a comment explaining the possible behaviors.